src/data.ts

```diff
@@ -1,13 +1,13 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
-
-export type Datum = any; // A Datum… who knows what it is?
+type ObjectDatum = Record<string, unknown>;
+export type Datum = ObjectDatum | Value;
```

Comment:
Does this reflect that Plot really accepts _any_ type of Datum? It's the role of an accessor to return a Value from that the Datum.

For one relatively legit example, here's an array of classes and functions and I want to plot their names and number of arguments.

`Plot.dotX([...Object.values(Plot)], { x: "length", y: "name" }).plot()
`